### PR TITLE
[ISSUE #6211][Test🧪] Add unit tests for PopProcessQueueInfo

### LINKED ISSUE #6211: [Test🧪] Add unit tests for PopProcessQueueInfo
## Prerequisites
- [ ] I have searched existing test issues

## Test Scope
**Module**: rocketmq-remoting  
**File**: `src/protocol/body/pop_process_queue_info.rs`  
**Current Coverage**: Unknown  
**Target Coverage**: 80%+

## Test Cases
The following test scenarios need to be implemented for `PopProcessQueueInfo`:

- [ ] Test struct field initialization
- [ ] Test all getter methods return correct values
- [ ] Test all setter methods update values correctly
- [ ] Test serialization to JSON
- [ ] Test deserialization from JSON
- [ ] Test serialization/deserialization roundtrip
- [ ] Test Default trait implementation (if derived)
- [ ] Test Clone trait implementation (if derived)
- [ ] Test Debug trait implementation (if derived)
- [ ] Test with zero and non-zero integer fields
- [ ] Test with boolean fields
- [ ] Test Display trait implementation (if implemented)

## Willing to contribute?
- [ ] I can submit a PR for this

### PR DESCRIPTION
**Pull request by oopscompiled**
### Which Issue(s) This PR Fixes(Closes)

- Fixes #6211

## Summary by CodeRabbit

* **Tests**
  * Added comprehensive unit tests for a core processing data structure covering initialization, accessors, setters, clone/copy semantics, display formatting, and edge-case values (zero, negative, and extreme numeric ranges). Tests are only included in test builds and do not affect runtime behavior or public APIs.

### CODE REVIEW

File: rocketmq-remoting/src/protocol/body/pop_process_queue_info.rs [hunk 1]
```diff
@@ -62,3 +62,56 @@ impl std::fmt::Display for PopProcessQueueInfo {
...
+    }
+
+    #[test]
+    fn pop_process_queue_clone() {
+        let queue: PopProcessQueueInfo = PopProcessQueueInfo::new(10, false, 123456789);
+        let cloned = queue.clone();
+        assert_eq!(cloned.wait_ack_count(), 10);
+        assert!(!cloned.droped());
+        assert_eq!(cloned.last_pop_timestamp(), 123456789);
+    }
```
[the remark this answers is not in the document]
Comment (oopscompiled) [reply rocketmq-remoting/src/protocol/body/pop_process_queue_info.rs:97]: fix
